Assignment5/Address.java

```diff
@@ -0,0 +1,18 @@
+
+import java.io.Serializable;
+
+public class Address implements Serializable {
+
+	String city;
```

Comment:
Class fields are typically private and they are exposed thru getters and setters.